### ISSUE TITLE
onPaymentMethodReceived is not called for paypal on 2.27

### ISSUE THREAD

**Issue by prestes** [braintree/braintree-web#215]
Since version 2.27, onPaymentMethodReceived is never called for paypal integration

```javascript
braintree.setup('{{ backendGeneratedBraintreeToken }}', 'paypal', {
    container: 'paypal-container',
    paymentMethodNonceInputField: 'formId'),
    onPaymentMethodReceived: function (obj) {
        alert(1); // never called
        // here we read obj.nonce and post to the backend to handle things
    },
    locale: 'it_it'
});
```

When doing the same in 2.26 it works perfectly

**Comment by crookedneighbor** [comment]
Thanks for the report. We're working on fixing the regression now.

**Comment by mrak** [comment]
We have released version `2.29.0` that addresses this issue. Thanks for reporting!
